### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get_all_tickers'

### ISSUE THREAD

**Issue by crack00r** [miti0/mosquito#24]
```
root@j314793:/mosquito# python3 mosquito.py --backtest --plot
setting_all_pairs
Traceback (most recent call last):
  File "mosquito.py", line 35, in <module>
    main(args)
  File "mosquito.py", line 8, in main
    engine = Engine(args, 'config.ini')
  File "/root/mosquito/core/engine.py", line 49, in __init__
    self.bot = Backtest(args, config_file)
  File "/root/mosquito/core/bots/backtest.py", line 21, in __init__
    super(Backtest, self).__init__(args, config_file)
  File "/root/mosquito/core/bots/base.py", line 27, in __init__
    self.pairs = self.process_input_pairs(self.config['Trade']['pairs'])
  File "/root/mosquito/core/bots/base.py", line 34, in process_input_pairs
    return self.exchange.get_all_tickers()
AttributeError: 'NoneType' object has no attribute 'get_all_tickers'
root@j314793:/mosquito#
```

**Comment by crack00r** [comment]
```
root@j314793:~/mosquito# python3 mosquito.py --paper
setting_all_pairs
Traceback (most recent call last):
  File "mosquito.py", line 35, in <module>
    main(args)
  File "mosquito.py", line 8, in main
    engine = Engine(args, 'config.ini')
  File "/root/mosquito/core/engine.py", line 52, in __init__
    self.bot = Paper(args, config_file)
  File "/root/mosquito/core/bots/paper.py", line 15, in __init__
    super(Paper, self).__init__(args, config_file)
  File "/root/mosquito/core/bots/base.py", line 27, in __init__
    self.pairs = self.process_input_pairs(self.config['Trade']['pairs'])
  File "/root/mosquito/core/bots/base.py", line 34, in process_input_pairs
    return self.exchange.get_all_tickers()
AttributeError: 'NoneType' object has no attribute 'get_all_tickers'

```

**Comment by miti0** [comment]
Thank you for reporting this - I definitely need to fix unit tests soon. I have pushed a fix [216d3b0](https://github.com/miti0/mosquito/commit/216d3b0186e4a30f39006eff93977a24f6e8756a). Please pull new changes.

btw: I will now focus on multi-currency strategy (mosquito), so hopefully we will get some positive results soon.
